futures-util/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "futures-util"
+version = "0.1.0"
+authors = ["Alex Crichton <alex@alexcrichton.com>"]
+license = "MIT/Apache-2.0"
+repository = "https://github.com/alexcrichton/futures-rs"
+homepage = "https://github.com/alexcrichton/futures-rs"
+documentation = "https://docs.rs/futures-core"
+description = """
+Common utilities and extension traits for the futures-rs library.
+"""
+
+[features]
+std = ["futures-core/std", "futures-channel/std", "futures-executor/std", "futures-sink/std"]
+default = ["std"]
+
+[dependencies]
+futures-core = { path = "../futures-core", version = "0.1.0" }
+futures-channel = { path = "../futures-channel", version = "0.1.0" }
+futures-executor = { path = "../futures-executor", version = "0.1.0" }
+futures-sink = { path = "../futures-sink", version = "0.1.0" }
```

Comment:
Ah now this would definitely be a good optional dependency (if you don't otherwise use sinks/channels/etc)

Reply:
Although in this case of a util crate it may be a bummer b/c "util" tends to be "wait but I want this to be easy"